secure_fw/partitions/crypto/crypto_check_config.h

```diff
@@ -118,4 +118,15 @@
 #error "CRYPTO_KEY_DERIVATION_MODULE_ENABLED enabled, but not all prerequisites (missing key derivation algorithms)!"
 #endif
 
+/* TODO check what is needed here */
```

Comment:
did you forget this TODO or would you like to keep it? :)

Reply:
Good catch, removed the TODO